parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveStringifier.java

```diff
@@ -24,6 +24,7 @@
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.NANOSECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.apache.parquet.util.Float16.toFloat;
```

Comment:
IMO, we should only import `import org.apache.parquet.util.Float16` and explicitly call `Float16.toFloat`